src/MainWindow.cpp

```diff
@@ -721,6 +721,20 @@ void MainWindow::populateTable()
         connect(ui->dataTable->selectionModel(), SIGNAL(currentChanged(QModelIndex,QModelIndex)), this, SLOT(dataTableSelectionChanged(QModelIndex)));
         connect(ui->dataTable->selectionModel(), &QItemSelectionModel::selectionChanged, [this](const QItemSelection&, const QItemSelection&) {
             updateInsertDeleteRecordButton();
+
+            const QModelIndexList& sel = ui->dataTable->selectionModel()->selectedIndexes();
+            if (sel.count() > 1 && sel.count() < Settings::getValue("databrowser", "complete_threshold").toInt()) {
+                int rows = sel.last().row() - sel.first().row() + 1;
+                int columns = sel.last().column() - sel.first().column() + 1;
+                double sum = 0;
+                for (const QModelIndex& index : sel) {
+                    QVariant data = m_browseTableModel->data(index, Qt::EditRole);
+                    sum += data.toDouble();
+                }
+                ui->statusbar->showMessage(QString("%1 rows, %2 columns. Sum: %3. Average: %4").arg(rows).arg(columns).arg(sum).
```

Comment:
This should be ```tr(...)``` to make it translatable, maybe even using the [plurals idiom](https://doc.qt.io/qt-5/i18n-source-translation.html#handling-plurals).

Reply:
Not the first time that I forget... :+1: 

About plurals, I'm intentionally discarding one-cell selections, because just clicking on a cell, selects it, but it still makes sense, because one of the sections could be singular. But then, we have to split the message in two so each one has its own %n?

What I find annoying about the plurals idiom is that it doesn't work for the untranslated version. Shouldn't Qt have designed a way to make it work only with the source code?

Reply:
Oh yeah, I missed the ```if``` statement checking for single cell selections. You're right then that we would have to make two messages out of this. Not sure if it's worth the effort at the moment. Let's just merge this anyway, we can always change it later :smile:

I'm not sure how you could make the plurals idiom work without any aid from a translation file. In English it seems rather simple but in other languages some words really change a lot depending on their number (like in Russian 1 год -> 1 year, 5 лет -> 5 years). Maybe they didn't want to make any assumptions on the base language of the application, but I don't know :smiley: